pages/api/policy-groups/[name]/assignments.js

```diff
@@ -75,6 +75,7 @@ export default async (req, res) => {
 
       const response = await post(
         `${rodeUrl}/v1alpha1/policies/${requestBody.policyVersionId}/assignments/${name}`,
+        requestBody,
```

Comment:
Was this not working suddenly? I've definitely seen assignments work through the UI before. 

The only fields a client needs to send are the policy version id and the group name, both of which come from the path parameters. So we can send the body, but it'll be overwritten

Reply:
For me, this doesn't work locally without a body.  I ended up just using `requestBody` here, but `{}` (empty body) also works.  But with no body, this always fails with a 400 from rode.  The specific error message I get is

```
{
  code: 3,
  message: "invalid character 'e' looking for beginning of value",
  details: []
}
```

Reply:
The same request with an empty body works fine in Postman also

Reply:
I think this may be a bug from some changes in #142. That added a third parameter to the `post` utility here for the access token. When auth is enabled, the body is being set to the access token value, the invalid 'e' is the from the token header `eyJhb...`. 

I think we should set this to null or an empty object instead, since the values are the body are overwritten

Reply:
Makes sense, I'll update it.